core/src/main/scala/org/apache/spark/status/AppStatusStore.scala

```diff
@@ -112,10 +112,13 @@ private[spark] class AppStatusStore(
     }
   }
 
-  def stageAttempt(stageId: Int, stageAttemptId: Int, details: Boolean = false): v1.StageData = {
+  def stageAttempt(stageId: Int, stageAttemptId: Int,
+    details: Boolean = false): (v1.StageData, Seq[Int]) = {
     val stageKey = Array(stageId, stageAttemptId)
-    val stage = store.read(classOf[StageDataWrapper], stageKey).info
-    if (details) stageWithDetails(stage) else stage
+    val stageDataWrapper: StageDataWrapper = store.read(classOf[StageDataWrapper], stageKey)
```

Comment:
you don't really need the type here (: StageDataWrapper)

Reply:
Done, thanks!